src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -348,6 +348,17 @@
      */
     serializedClassDetails,
 
+    /**
+     * The class for the list containing the {@code @see} tags of an element.
+     */
+    seeList,
+
+    /**
+     * The class for the list containing the {@code @see} tags of an element
+     * when some of the tags have longer labels.
+     */
+    seeListLong,
+
```

Comment:
The list is alpha-sorted in each group, suggesting that these new entries should go before `serializedClassDetails`.